runtime/queries/typst/highlights.scm

```diff
@@ -0,0 +1,77 @@
+(call
+  item: (ident) @function)
+(call
+  item: (field field: (ident) @function.method))
+(tagged field: (ident) @tag)
+(field field: (ident) @tag)
+(comment) @comment
+
+; CONTROL
+(let "let" @keyword.storage.type)
```

Comment:
having `(let "let"`... looks weird, is this really necessary?
i would assume, that the `let` node already captures it.
The same goes with all the others.